boards/v2/arm/mps3_an547/mps3_an547_ns.yaml

```diff
@@ -4,7 +4,7 @@
 # SPDX-License-Identifier: Apache-2.0
 #
 
-identifier: mps3_an547_ns
+identifier: mps3/an547/ns
```

Comment:
hmm, are we changing how boards are named?

Reply:
the proposal is to name a board, as the board.
And then specify / differentiate between (sips) / socs / cpuclusters / variants by separating those with a `/`.

For example, building for the cpuapp core on nRF5340 SoC for nrf5340dk then becomes: `nrf5340dk/nrf5340/cpuapp`.

This also means there will be fewer boards in the board list, as nrf5340dk will be just one board instead of 3 boards.
Will be presenting details at the arch wg, and it will be detailed described in upcoming docs.